### DISCUSSION TITLE
SSH into the controller

### DISCUSSION THREAD

**Discussion by williamgeurts** [hassio-addons/addon-unifi#413] [Q&A] [answered]
Hi,
I want to a switch port POE on and off. I cannot get this working using API's. So I want to move to plan B. When I am able too ssh into the controller I found the commands to switch POE on and off. But SSH is not possible on the HomeAssistant Add-on-unifi. Anyone an idee how to get around this?

William

**Comment by frenck** [comment, marked as answer]
There is no workaround for that; there is no SSH available.

If you want to do things like that, then this add-on is not the right fit for you. Consider installing/using the UniFi Network Application in a different way.

../Frenck

**Reply by refesler** [reply]
It looks like one might be able to achieve what he wanted by sshing to HA after disabling protection in Advanced SSH (thank you also Frenck).  At that point you would have some access to the docker filesystems, anyway.

**Comment by williamgeurts** [comment]
Hi Frenck,

Thanks for the quick answer, back to trying to get the API working.

-- 
Met vriendelijke groet,

William Geurts
